src/app-utils/make-prefs-migration-script.xsl

```diff
@@ -217,4 +233,9 @@
   <df:dateformat><df:name>locale</df:name><df:index>4</df:index></df:dateformat>
 </df:dateformats>
 
+<ct:calendartypes>
+    <ct:calendartype><ct:name>gregorian</ct:name><ct:index>0</ct:index></ct:calendartype>
+    <ct:calendartype><ct:name>jalali</ct:name><ct:index>1</ct:index></ct:calendartype>
+</ct:calendartypes>
+
 </xsl:stylesheet>
```

Comment:
There is no need to add the calender type as a migratable pref. This feature is only used to migrate settings from gconf to dconf. Gconf was used until gnucash 2.4 so only preferences that already existed should be in here.